odilia/src/main.rs

```diff
@@ -105,10 +113,9 @@ async fn speak(
 #[tracing::instrument(ret)]
 async fn doc_loaded(
 	loaded: LoadCompleteEvent,
-	Speech(ssip): Speech,
-) -> Result<Vec<Command>, odilia_common::errors::OdiliaError> {
```

Comment:
-> impl IntoCommand would be simpler perhaps, depends if the type checker is smart enough for that one.

Reply:
Great plan!